modules/boilerplates/repl.go

```diff
@@ -0,0 +1,6 @@
+package main
+import ("fmt")
+
+func main() {
+  fmt.Println("Hello World! whats up")
```

Comment:
Could you remove the ` whats up` at the end here?

Reply:
> 

@DamianRivas Makes sense, updated!